cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -175,7 +230,7 @@ def is_vm_analysis_finished(vm_name):
     except:
         return False
     # throw exception if status is error
-    if 'Error' in sel.get_attribute(sel.element('//td/img', root=el), 'title'):
+    if 'Error' in sel.element('.//td/img', root=el).get_attribute('title'):
```

Comment:
Is there a problem with the sel.get_attribute function here? It should do the same and we use this to abstract away interacting directly with selenium objects.